tap_hubspot/hubspot.py

```diff
@@ -237,7 +236,7 @@ def get_deals_v2(
         )
 
         for chunk in chunker(gen, 10):
-            ids: List[str] = [deal["id"] for deal in chunk]
+            ids: List[str] = [contact["id"] for contact in chunk]
```

Comment:
could you do a fixup for this one to clean up the history?

Reply:
This has been fixed in a later commit